contrib/gomodule/redigo/redigo_test.go

```diff
@@ -7,11 +7,11 @@ import (
 	"testing"
 
 	"github.com/gomodule/redigo/redis"
-	"github.com/stretchr/testify/assert"
 	"github.com/signalfx/signalfx-go-tracing/ddtrace/ext"
 	"github.com/signalfx/signalfx-go-tracing/ddtrace/mocktracer"
 	"github.com/signalfx/signalfx-go-tracing/ddtrace/tracer"
 	"github.com/signalfx/signalfx-go-tracing/internal/globalconfig"
+	"github.com/stretchr/testify/assert"
```

Comment:
usually, the pattern is to have 3 sections separated by a blank line:
- standard library packages
- 3rd party (external) packages
- current repo packages

I think it was first introduced by `goimports`. See: https://github.com/golang/tools/blob/master/cmd/goimports/goimports.go#L55

Personally, I like https://github.com/mvdan/gofumpt
It is possible to configure VS Code's gopls to use it ;) 
See: https://github.com/golang/vscode-go/blob/master/docs/settings.md#goformattool
This is my config I tend to use: https://github.com/golang-templates/seed/blob/main/.vscode/settings.json

It is also possible to configure `golangci-lint` to check for such issues.

Reply:
I never pay attention to this when writing go because gofmt/goimports are sooo awesome... until they aren't. fixing.